src/cpp/utils/IPFinder.cpp

```diff
@@ -132,7 +131,7 @@ bool IPFinder::getIPs(std::vector<info_IP>* vec_name, bool return_loopback)
 
     if (getifaddrs(&ifaddr) == -1) {
         perror("getifaddrs");
-        exit(EXIT_FAILURE);
+	return false;
```

Comment:
The indentation doesn't seem to align and also uses tabs instead of spaces.

Same below.

Reply:
Yep, good call.  Fixed now in 8e75a59.